modules/mod_login/tmpl/default.php

```diff
@@ -91,8 +91,8 @@
 		<?php if (PluginHelper::isEnabled('system', 'remember')) : ?>
 			<div class="mod-login__remember form-group">
 				<div id="form-login-remember-<?php echo $module->id; ?>" class="form-check">
-					<label class="form-check-label">
-						<input type="checkbox" name="remember" class="form-check-input" value="yes">
+					<label for="form-check-label">
+						<input type="checkbox" name="remember" id="form-check-label" value="yes">
```

Comment:
why do you call an input a label? makes no sense call it input

Reply:
This file is not a part of the Keyboard Shortcut Plugin, even I am confused about how it gets changed.